openslides/__main__.py

```diff
@@ -135,7 +135,7 @@ def start(args):
     execute_from_command_line(['manage.py', 'migrate'])
 
     if not args.no_browser:
-        start_browser('http://0.0.0.0:8000')
+        start_browser('http://127.0.0.1:8000')
```

Comment:
I am not sure if we should use `localhost` here because this uses the local socket instead of a TCP/IP connection. http://stackoverflow.com/questions/3715925/localhost-vs-127-0-0-1